app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -196,9 +196,13 @@ class NavigationOverlayFragment : Fragment() {
         val tintDrawable: (Drawable?) -> Unit = { it?.setTint(ContextCompat.getColor(context!!, R.color.photonGrey10_a60p)) }
         navUrlInput.compoundDrawablesRelative.forEach(tintDrawable)
 
+        registerForContextMenu(channelsContainer)
         canShowUnpinToast = true
 
-        pinnedTileChannel = defaultChannelFactory.createChannel(context!!, view as ViewGroup, R.id.pinned_tiles_channel)
+        defaultChannelFactory = createChannelFactory().apply {
+            pinnedTileChannel = createChannel(context!!, view as ViewGroup, R.id.pinned_tiles_channel)
```

Comment:
fwiw, if pinnedTileChannel's Observables where attached in onCreateView rather than onStart, we wouldn't need to keep a reference to it outside the scope of that method so we would be less likely accidentally create a memory leak here.